packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -829,7 +933,7 @@ export class Project extends ProjectBase {
 
     if ((sourceType === CODEPIPELINE_TYPE || artifactsType === CODEPIPELINE_TYPE) &&
       (sourceType !== artifactsType)) {
-        throw new Error('Both source and artifacts must be set to CodePipeline');
+      throw new Error('Both source and artifacts must be set to CodePipeline');
```

Comment:
If you're changing this, please double-intend the line above, so they're not on the same level.